### ISSUE TITLE
Trivial fix: vmbetter docs

### ISSUE THREAD

**Issue by mkunz7** [sandia-minimega/minimega#639]
```cp -r misc/vmbetter_configs/generic_vm_overlay misc/vmbetter_configs/wireshark_overlay```
should be
```cp -r misc/vmbetter_configs/default_overlay misc/vmbetter_configs/wireshark_overlay```

```
$ du -a misc/vmbetter_configs/wireshark_overlay/
4       generic_vm_overlay/init
4       generic_vm_overlay/etc/rc.local
8       generic_vm_overlay/etc
16      generic_vm_overlay/
```

rc.local doesnt exist in vmbetter_configs anymore, perhaps it needs to be added back in I just copied mine from my server. It doesn't look like we made any changes to it, so perhaps we can just get rid of /etc/* lines above.

On ubuntu server 16.04 you also need debootstrap genisoimage extlinux from apt-get

```vm launch 1```
needs to be vm launch kvm

**Comment by mkunz7** [comment]
https://github.com/sandia-minimega/minimega/pull/642

**Comment by djfritz** [comment]
Can we close this now? @mkunz7

**Comment by mkunz7** [comment]
The push didn't update to minimega.org, is there something special we have to do there?

**Comment by jcrussell** [comment]
`http://minimega.org/` is only updated when we push a release, `http://tip.minimega.org/` is built from master.

**Comment by mkunz7** [comment]
tip doesn't have the changes from the pr either

**Comment by jcrussell** [comment]
This all shows up fine when I run `minidoc` locally. We'll fix the external website in #704.
